packages/fxa-settings/src/styles/ctas.scss

```diff
@@ -14,32 +19,33 @@
   }
 }
 
-@screen mobileLandscape {
-  .cta-neutral {
-    @apply text-xs py-1 px-5 mt-0 rounded-sm;
-  }
-}
-
-.cta-neutral-lg {
-  @apply block w-full text-center bg-grey-10 border border-grey-200 text-base mr-4 rounded mt-0 py-2;
+.cta-primary {
+  @apply bg-blue-500 border-blue-600 text-white;
 
   &:hover {
-    @apply border-grey-200 bg-grey-100 text-grey-400;
+    @apply bg-blue-600 border-blue-600;
   }
 
   &:active {
-    @apply border-grey-400 bg-grey-100 text-grey-400;
+    @apply bg-blue-800 border-blue-800;
   }
 }
 
-.cta-primary {
-  @apply bg-blue-500 py-2 text-white border border-blue-600 rounded;
+.cta-lg {
+  @apply text-base;
+}
```

Comment:
Since `text-base` is default, do we even need this class?

I don't mind these changes, but I did notice we're missing a `:focus` state here. Would you like to add that perhaps? 😬 (not needed though)

Reply:
> we're missing a :focus state here

That's what the primary/neutral classes would be used for (which have focus states) -- one class for sizing, one for style. Ideally we could bake the "regular" size into both the style classes and only use cta-lg when we wanted them to be bigger, but because the regular size uses breakpoints and cta-lg doesn't it means we'd need to introduce breakpoints for it as well just to take selector priority, and it becomes a slippery slope from there. I hope that makes sense. I'm happy to demo if you'd like 😅

Reply:
Good catch for both of these :)